### DISCUSSION TITLE
MacOS-12/13 Building is Deprecated ⚠️

### DISCUSSION THREAD

**Discussion by aricloverEXTRA** [arichornlover/uYouEnhanced#763] [Q&A] [answered]
Hello everyone. I have made this discussion to let everyone know that MacOS-12 is being deprecated. And there is plans on updating the building to MacOS-13.

if you want more info check https://github.com/actions/runner-images/issues/10721

MacOS-12 will be fully unsupported by December 3rd, 2024.

**Comment by aricloverEXTRA** [comment, marked as answer]
uYouEnhanced now builds on **macos-13**
everything should work as intended since I have fixed it.

**Reply by mehargiri** [reply]
I just tried to build uYouEnhanced using the latest commit but it is still building on macOS-12, specifically macos 12.7.6.

Could you take a look at the .github/workflows/buildapp.yml file and make the necessary changes?

I updated the macOS version to macos-13 in my forked repo and tried to build uYouEnhanced using the latest commit but I got an error on the 'Fix compiling & Build Package' stage of the build action.

I am attaching an image of the error for your reference. 

![image](https://github.com/user-attachments/assets/cc83b261-41d4-4f4c-8a3a-df9c91b8d4eb)

**Reply by aricloverEXTRA** [reply]
That happens on macos-13. I’ll see if I can resolve it.
